gap/projective/almostsimple.gi

```diff
@@ -1328,7 +1328,7 @@ FindHomMethodsProjective.SporadicsByOrders := function(ri,G)
   ordersseen := [];
   count := 0;
   gens := GeneratorsOfGroup(G);
-  limit := 120+Length(gens);
+  limit := 220+Length(gens);
```

Comment:
But where do these magic numbers come from? Are they based on a paper, or did someone just pull them from their a** ?

I'll merge it but I don't like it. Perhaps working on #157 will allow us to improve things.

Reply:
I just increased the values until SporadicsByOrders didn't rule out the wrong or too few groups. 

SporadicsByOrders stores for each sporadic group the most common element orders and their probabilities. During recognition of a group `G`, if after enough tries a common element order of a sporadic group `T` was not seen at all, then it is ruled out that `G` is isomorphic to `T`. For some types of sporadic groups that seems to happen too often. So maybe just some of the probabilities are incorrect. I can start by looking into the most common orders of the Suzuki group "Suz".

The only reference I found inside `recog/almostsiple.gi` that could maybe help here is `<Cite Key="KS09"/>` i.e. "Large element orders and the characteristic of Lie-type simple groups" by Kantor and Seress. Maybe it gives hints or references on how to choose the parameters to do the sporadic case.